subgraphs/compound-forks/tectonic/src/constants.ts

```diff
@@ -0,0 +1,67 @@
+import { Address } from "@graphprotocol/graph-ts";
+import { cTokenDecimals } from "../../src/constants";
+import { TokenData } from "../../src/mapping";
+
+//////////////////////////////
+/////     Addresses      /////
+//////////////////////////////
+
+export let comptrollerAddr = Address.fromString(
+  "0xb3831584acb95ED9cCb0C11f677B5AD01DeaeEc0"
+);
+
+export const nativeToken = new TokenData(
+  Address.fromString("0x0000000000000000000000000000000000000000"),
+  "Cronos",
+  "CRO",
+  18
+);
+
+export const tCROToken = new TokenData(
```

Comment:
All the tokens below this can be removed and will be created in `compound-forks/src/mapping.ts` or you can utilize the ERC20.json to grab the data for this. future proofing